typescript/package.json

```diff
@@ -1,29 +1,25 @@
 {
-  "name": "wot-typescript-definitions",
-  "version": "0.8.0-SNAPSHOT.16",
-  "description": "TypeScript definitions for the W3C WoT Scripting API",
-  "author": "W3C Web of Things Working Group",
-  "license": "W3C-20150513",
-  "repository": "https://github.com/w3c/wot-scripting-api/tree/master/typescript",
+  "name": "wot-typescript",
+  "version": "1.0.0",
```

Comment:
What are the implications of this version? 
I assume we need to update every time to be able to publish on NPM?

Reply:
Hence, I suggest to change to `0.0.1`

Reply:
@relu91 OK, also mark as private

Reply:
Solved!